presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -3,9 +3,20 @@ Web UI
 ======
 
 Presto provides a web-based user interface (UI) for monitoring a Presto cluster
-and managing queries. The web UI is accessible on the Presto coordinator via
+and managing queries. The Web UI is accessible on the coordinator via
 HTTP/HTTPS, using the corresponding port number specified in the coordinator
-:ref:`config_properties`.
+:ref:`config_properties`. It can be configured with :ref:`specific related
+properties <web-ui-properties>`.
+
+The Web UI requires users to log in on the initial page. A default Presto
+installation has no authorization and authentication enabled and you can provide
+any username to log in successfully. Best practice is to use the same username,
+when connecting to other systems in your organization, including running queries
+on Presto with the CLI or some other application.
+
+When the server is configured to use HTTPS/TLS, a username/password
```

Comment:
Let's just say HTTPS since that's what we use elsewhere in the documentation.

Reply:
We can simplify this to
```
When the server is configured to use HTTPS, a :doc:`/develop/password-authenticator`
such as :doc:`LDAP </security/ldap>` must be configured in order to use the Web UI.

Reply:
adopted but added password auth and specific wording for links